BankexWallet/InterfaceElements/DesignElements/Onboarding/RoundedButton.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  RoundedButton.swift
+//  BankexWallet
+//
+//  Created by Георгий Фесенко on 01/06/2018.
+//  Copyright © 2018 Alexander Vlasov. All rights reserved.
+//
+
+import UIKit
+
+class RoundedButton: UIButton {
+    override func awakeFromNib() {
```

Comment:
This will work correctly only if 
- you draw the button inside storyboard
- you draw it with correct height and don't need to change it 

So either let's do fully working special class, or let's continue to write this 1 line of round corners in each place we really need it

Reply:
I've deleted this file and did it in runtime attributes